s4e-web/src/app/views/settings/email-list-validator.utils.ts

```diff
@@ -0,0 +1,15 @@
+import { AbstractControl } from '@angular/forms';
+
+const COMMA_SEPARATED_EMAILS_EXPRESSION = /((,? ?)((\.?)[a-zA-Z0-9_]+)+@((\.?)[a-zA-Z0-9_]+)+)+/i;
```

Comment:
Why not use a ready regex, for example from http://emailregex.com/? The HTML5 W3C version seems quite ok.

Reply:
It's almost the same as `[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+` on-site you just send me, but more human readable (, because without chars in ASCII HEX values).

Reply:
Okay, just handle the following cases correctly (input -> result):
"@" -> fail
"d@d" -> ok
" d@d    " -> ok (I'd strip all the whitespace)
" d@d, d@d " -> ok
" d@d, d " -> fail
"d@d,d" -> fail
"d,d@d" -> fail
"s.o_m-e+@mail-0.pl" -> ok
"@d@" -> fail
"d@d@" -> fail
"d@d@d" -> fail
"d" -> fail
"" -> ok (I guess, although I'm not sure about that one)
Make a test for that.

A thing you probably plan to handle is also that it outputs now instead of a sensible message:
![image](https://user-images.githubusercontent.com/583197/96994309-d3bc2800-152c-11eb-84df-edc61016d3fe.png)